### DISCUSSION TITLE
status line and "visual"

### DISCUSSION THREAD

**Discussion by wf5w** [LunarVim/LunarVim#3370] [Q&A] [answered]
Hey y'all:

When I go into visual mode, the status line doesn't show that I am in visual mode (or which type of visual mode i am in).
How do I add that?

Awesome job with LVIM, I am really enjoying it.

Thanks.
Jerry

**Comment by wf5w** [comment, marked as answer]
Ok. I think I found the answer. Change the lualine style from lvim to default. Now I know what mode I am in.


diff --git a/lua/lvim/core/lualine/init.lua b/lua/lvim/core/lualine/init.lua
index 0ee35c0..a4aa80c 100644
--- a/lua/lvim/core/lualine/init.lua
+++ b/lua/lvim/core/lualine/init.lua
@@ -2,7 +2,7 @@ local M = {}
 M.config = function()
   lvim.builtin.lualine = {
     active = true,
-    style = "lvim",
+    style = "default",
     options = {
       icons_enabled = nil,
       component_separators = nil,

**Reply by zenofile** [reply]
To retain the lvim lualine style and just insert the mode, I use `lvim.builtin.lualine.sections.lualine_a = { "mode" }` instead.

**Reply by wf5w** [reply]
zenofile: where would I add that, exactly?
1. there is init.lua somewhere?
2. in styles.lua it refers to components.mode
3. in components.lua it looks like it returns an empty? function.

So where would I put that statement without breaking the model?
I don't really know the lua language all that well.

Thanks in advance,

Jerry

**Reply by zenofile** [reply]
> zenofile: where would I add that, exactly?

In your `config.lua`, see the first paragraph of https://www.lunarvim.org/docs/configuration if you don't have one already.
